schema.sql

```diff
@@ -69,7 +69,11 @@ CREATE TABLE genotypes (
        quality TEXT,
 
        -- Cycledash-derived data
-       "annotations:gene_names" TEXT,
+       -- These are from the varcode annotation worker
+       "annotations:gene_name" TEXT,
```

Comment:
@armish You only have a single gene if you're picking the top priority annotation for each variant. 

You could do this to get the top effect for each distinct gene:

```
for gene_name, gene_effects in variant.effects().groupby_gene_name().items():
    top_gene_effect = gene_effects.top_priority_effect()
```

but you'll have to then deal with all sorts of weird non-coding gene types.